lib/ClangImporter/ImportType.cpp

```diff
@@ -806,16 +806,20 @@ namespace {
     }
 
     ImportResult VisitEnumType(const clang::EnumType *type) {
-      auto clangDecl = type->getDecl();
+      auto clangDecl = type->getDecl()->getDefinition();
+      if (!clangDecl) {
+        // FIXME: If the enum has a fixed underlying type, can we use that
+        // instead? Or import it opaquely somehow?
```

Comment:
(outside the scope of this PR) what do you mean by import it opaquely and how would that work? Wouldn't this have the same problem as forward declared structs? As for size, all enums have a size I thought, whether implicit or with explicit type specifier.